### ISSUE TITLE
Flaky `test_vfio_user` on aarch64 worker

### ISSUE THREAD

**Issue by likebreath** [cloud-hypervisor/cloud-hypervisor#5366]
The `test_vfio_use` has been failing multiple times from #5364 and #5362, with the following error:

```
thread 'common_parallel::test_vfio_user' panicked at 'assertion failed: exec_host_command_status(\"/usr/local/bin/spdk-nvme/rpc.py nvmf_create_transport -t VFIOUSER\").success()', tests/integration.rs:6408:9
```

**Comment by likebreath** [comment]
@michael2012z @MrXinWang Can you please take a look at the issue? The failure itself looks not related to Cloud Hypervisor, as the error is from spdk-nvme RPC calls. Thanks a lot.

**Comment by likebreath** [comment]
For the record, both #5364 and $5352 took 3 runs to pass the `test_vfio_user` on aarch64 worker. Let's keep an eye on it for a bit longer.

**Comment by michael2012z** [comment]
Today I logged in the CI server and found some hanging docker containers (for a few days). It may be related.
I cleaned the pending containers. Let's monitor for some days.

**Comment by likebreath** [comment]
This should be addressed by #6032.
